CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## UNRELEASED
+
+- Linter now flags the presence of webpack and explains options.
```

Comment:
Let's include a caveat that it looks for the default config file. It's possible to have the config file named differently.